### DISCUSSION TITLE
Uno Platform +WinUI3/Winappsdk windows targeted app package and distribution

### DISCUSSION THREAD

**Discussion by SrinivasPam** [unoplatform/uno#10543] [Q&A]
Hi Team, we have developed a desktop app with Windows Head using  Uno Platform +WinUI3/Winappsdk. Now, we want to package and distribute the app to end users so that user can install and use it. Can you please guide us on what is the best way to do it. we are not considering publishing to windows App store at the moment. 

We are almost close to release the app however we are stuck here. We have gone through few docs but its not really helpful as most the docs said package needs to be signed using code signing certificate from verified CA. 

Any help would be greatly appreciated as we are blocked here now. 

**Evironment Snip**:
![image](https://user-images.githubusercontent.com/110016450/203471813-ba3faf5b-ebe3-4e99-8861-ca60f54e6c92.png)

**Running App Snip**
![image](https://user-images.githubusercontent.com/110016450/203472407-1229fccc-36d4-4a59-b795-778e4ec0cb13.png)

**Comment by jeromelaban** [comment]
Deploying apps to the Windows store is [handled by Microsoft's documentation](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/deploy-packaged-apps). If you have additional questions about WinAppSDK, it's best to ask on the [official repository](https://github.com/microsoft/WindowsAppSDK).
